src/hypertrace/agent/init/__init__.py

```diff
@@ -20,11 +20,10 @@
 
 class AgentInit:  # pylint: disable=R0902,R0903
     '''Initialize all the OTel components using configuration from AgentConfig'''
-    def __init__(self, agent: AgentConfig, init_console_only: bool = False):
+    def __init__(self, agentConfig: AgentConfig, init_console_only: bool = False):
```

Comment:
Should this be following the naming conventions? Not sure if lint catched this.

Reply:
An oversight. pylint seemed to miss it. It is now called agent_config.